src/services/Notion.ts

```diff
@@ -0,0 +1,50 @@
+import { Client } from '@notionhq/client';
+import { Config } from '../types';
+
+/** Class representing a interaction with Notion. */
+export class Notion {
```

Comment:
I propose to develop a common interface for services. be it Notion, Google Docs or whatever. this interface should contain an initialization function and a method for sending information, as you did. and when initializing the widget, the required interface will be selected, depending on the config.